packages/react-ui-core/test/google-maps-mock.js

```diff
@@ -0,0 +1,1055 @@
+// Found at: https://github.com/prasathmani/google-maps-mock-api
```

Comment:
We'll need to clean this up and either decide to use this file or the logic we currently have in `test/shim.js`.